File: 3.TimeComplexity - PermMissingElem/answer.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A):
    # write your code in Python 3.6
    A.sort()
    if(len(A) == 0):
        return 0
    ans = 1
    for i in range(1, len(A)):
        if(A[i] != A[i-1]):
            ans += 1 
    
    return ans
    pass

File: 3.TimeComplexity - PermMissingElem/answer.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
    // write your code in JavaScript (Node.js 8.9.4)
    let N = A.length + 1
    let partialSum = (N *(N + 1))/2
    const sum = A.reduce((a, b) => a + b, 0);
    
    return partialSum - sum
}
